### ISSUE TITLE
Temporal Consistency Constraint

### ISSUE THREAD

**Issue by agoodge** [seominseok0429/Data-driven-Precipitation-Nowcasting-Using-Satellite-Imagery#4]
Hi,

Thank you for the great work. I cannot find where the temporal consistency constraint is implemented in this code. I only see the MSELoss being applied in your training function. Please could you share the usage of the temporal consistency constraint?

Thanks!

**Comment by seominseok0429** [comment]
Please refer to the following GitHub repository for the loss function: https://github.com/Lupin1998/SimVPv2.
We initially used this loss function, but removed it from our implementation due to the significantly increased training time.
